### PR TITLE
update copy on restart page

### LINKED ISSUE #1386: Update style and copy on 'restart required' modal
Had a meeting with Winston today and he had some recommendations on this page. Current implementation: https://vid.me/4Mq9
- Increased font size on the 'Restart' CTA. Maybe bring it to the top.
- Add an 'almost done' implication to the strings.

Current strings ([ref](https://github.com/mozilla/testpilot/blob/master/locales/en-US/app.ftl#L146))
- `restartRequiredHeader` = Test Pilot Installed!
- `restartRequiredFromLanding` = Restart Firefox to choose your features.
- `restartRequiredFromExperiment` = Restart Firefox to enable {$experimentTitle}.

@johngruen to confirm strings (and maybe open PR).

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #1386

### CODE REVIEW

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -143,9 +143,9 @@ shareCopy = Copy
 
 eolMessage = <strong>This experiment is ending on {$completedDate}</strong>.<br/><br/>After then you will still be able to use {$title} but we will no longer be providing updates or support.
 
-restartRequiredHeader = Test Pilot Installed!
-restartRequiredFromLanding = Restart Firefox to choose your features.
-restartRequiredFromExperiment = Restart Firefox to enable {$experimentTitle}.
+restartRequiredSubHeader = Almost done . . .
```
Comment (chuckharmston) [inline locales/en-US/app.ftl:146]: Can we use `&hellip;` for this?

File: frontend/src/app/containers/Restart.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default class Restart extends React.Component {
       <div className="full-page-wrapper space-between">
         <Header forceHideSettings={ false } />
         <div className="centered-banner restart-message">
-          <h2 data-l10n-id="restartRequiredHeader">Test Pilot Installed!</h2>
+          <p data-l10n-id="restartRequiredSubHeader">Almost done . . s.</p>
```
Comment (chuckharmston) [inline frontend/src/app/containers/Restart.js]: This doesn't match the string in the localization file. Also, let's use `&hellip;` here, too.

**Review by chuckharmston — CHANGES_REQUESTED** [review]
Looks good other than two small comments.

File: locales/en-US/app.ftl (hunk 1 above)
Comment (johngruen) [reply locales/en-US/app.ftl:146]: @chuck Technically, the &hellip; character entity is not correct punctuation in this case: 

http://www.thepunctuationguide.com/ellipses.html

File: frontend/src/app/containers/Restart.js (hunk 2 above)
Comment (johngruen) [reply frontend/src/app/containers/Restart.js]: Fixed this % hellip inclusion.

**Review by chuckharmston — APPROVED** [review]

**Pull request merged**